src/table/TableBodyWrapper.jsx

```diff
@@ -26,12 +26,13 @@ const useStyles = makeStyles({
     },
   },
   srOnly: {
-    position: 'absolute',
-    left: '-10000px',
-    top: 'auto',
-    width: '1px',
+    clip: 'rect(0 0 0 0)',
+    clipPath: 'inset(50%)',
```

Comment:
Yes, I think this is the recommended way for styling the screen reader texts since we have a good support for `clip` and semi-good support for `clip-path` properties.

the previous version uses the `left: -SOME_LARGE_INT` that was not quite good in my opinion.